examples/basics/plotting/plot.py

```diff
@@ -35,25 +35,21 @@
         l.update_gl_state(depth_test=False)
 
 labelgrid = fig[0, 0].view.add_grid(margin=10)
-hspacer = vp.Widget()
-hspacer.stretch = (6, 1)
-labelgrid.add_widget(hspacer, row=0, col=0)
 
 box = vp.Widget(bgcolor=(1, 1, 1, 0.6), border_color='k')
-labelgrid.add_widget(box, row=0, col=1)
+box_widget = labelgrid.add_widget(box, row=0, col=1)
+box_widget.width = 90
+box_widget.height = 120
 
-vspacer = vp.Widget()
-vspacer.stretch = (1, 2)
-labelgrid.add_widget(vspacer, row=1, col=1)
+bottom_spacer = vp.Widget()
+labelgrid.add_widget(bottom_spacer, row=1, col=0)
 
 labels = [vp.Label('n=%d' % plot_nvals[i], color=colors[i], anchor_x='left')
           for i in range(len(plot_nvals))]
 boxgrid = box.add_grid()
 for i, label in enumerate(labels):
-    boxgrid.add_widget(label, row=i, col=0)
-hspacer2 = vp.Widget()
-hspacer2.stretch = (4, 1)
-boxgrid.add_widget(hspacer2, row=0, col=1)
+    label_widget = boxgrid.add_widget(label, row=i, col=0)
+
```

Comment:
try resizing the plot to something tiny. I get an error. We should probably make the plotting area of the widget have min size (1, 1) instead of (0, 0). It's a silly size anyway, but that way we might avoid an error.